README.md

```diff
@@ -84,14 +84,20 @@ jobs:
       contents: write
       issues: write
       pull-requests: write
+      # optional: `id-token: write` permission is required if `provenance: true` is set below
+      id-token: write
     steps:
       - uses: nearform-actions/optic-release-automation-action@v4
         with:
-          npm-token: ${{ secrets.NPM_TOKEN }}
-          optic-token: ${{ secrets.OPTIC_TOKEN }}
           commit-message: ${{ github.event.inputs.commit-message }}
           semver: ${{ github.event.inputs.semver }}
           npm-tag: ${{ github.event.inputs.tag }}
+          # optional: set this secret in your repo config for publishing to NPM
+          npm-token: ${{ secrets.NPM_TOKEN }}
+          # optional: set this secret in your repo config for 2FA with Optic
```

Comment:
These lines were described as optional previously in the bullets below:

> _(Optional)_ You can define Npm and Optic tokens in GitHub secrets for each user that will receive the OTP. This is required only in case you want to publish to Npm.

...so now there's more optional config, I've moved all the optional lines together with comments